src/style.css

```
@@ -0,0 +1,34 @@
+.body {
+
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 13px;
+
+}
```

delete this unnecessary spaces everywhere in this file

```suggestion
.body {
  font-family: Arial, Helvetica, sans-serif;
  font-size: 13px;
}
```